codemagic.yaml

```diff
@@ -4,8 +4,12 @@ workflows:
     max_build_duration: 60
     environment:
       xcode: 16.1
+      vars:
+        XDG_STATE_HOME: /tmp/
       groups:
         - github_credentials
+    artifacts:
+      - /tmp/tuist/logs/*.log
```

Comment:
This can be useful for debugging Tuist outside of the `xcodebuild` logs, which are automatically uploaded to the server.

Reply:
We should upload the logs to the server in the future.